src/views/artefact-editor/roi-layer.vue

```diff
@@ -39,10 +46,16 @@ export default class RoiLayer extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '../../assets./styles/_variables.scss';
```

Comment:
My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.